### DISCUSSION TITLE
Adding S2 without reincluding

### DISCUSSION THREAD

**Discussion by brumlemann** [hassio-addons/addon-zwavejs2mqtt#482] [Q&A]
Hi

Unfortunately I added a lot of in-wall dimmer units without specifying S2.  Is it possible to change this without un-incluing and re-including? Annoying prospect to go around with a screwdriver to all my in-wall dimmers. 

Regards
M

PS: ChatGPT told me that it was possible to do this with the HA Z-Wave JS UI plug-in but the recipe for doing it doesn't match what I see. it tells me to go to the Z-Wave JS UI Devices sections (Control Panel?) look for the ""More Options" button (three vertical dots) in the top right corner and select "Edit Node Configuration" from the dropdown menu".... Is this an artifact of the cutoff date of ChatGPT? Talking about an older version than the one I have installed?

PPS I went back and asked ChatGP which version i was basing it's "knowledge" on and it replied  0.1.43.

**Comment by Ltek** [comment]
what is the solution/answer here?
